packages/components/fields/text-field/docs/description.md

```diff
@@ -0,0 +1 @@
+A controlled text input component for single-line strings with validation states.
```

Comment:
I think one line description can be added to the component's package.json since the script checks that and if present it will just that one.